mittab/libs/outround_tab_logic/pairing.py

```diff
@@ -0,0 +1,109 @@
+import math
+
+from mittab.apps.tab.models import *
+
+from mittab.libs.outround_tab_logic.checks import have_enough_judges, \
+    have_enough_rooms
+from mittab.libs.outround_tab_logic.bracket_generation import gen_bracket
+
+from mittab.libs.tab_logic import have_properly_entered_data
+from mittab.libs import errors
+
+
+def good_to_go(num_teams):
+    # Check if people can math properly -- ie is it a power of two
+    if not math.log(num_teams, 2) % 1 == 0:
+        # ERROR: Not a power of 2, someone can't math
```

Comment:
What about partial outrounds?

Reply:
Currently doesn’t support, but I’ll get on that.

Reply:
Supports partials now!